logbook-httpclient5/src/test/java/org/zalando/logbook/httpclient5/LogbookHttpExecHandlerTest.java

```diff
@@ -0,0 +1,46 @@
+package org.zalando.logbook.httpclient5;
+
+import org.apache.hc.client5.http.classic.methods.HttpGet;
+import org.apache.hc.client5.http.classic.methods.HttpPost;
+import org.apache.hc.client5.http.impl.classic.CloseableHttpClient;
+import org.apache.hc.client5.http.impl.classic.HttpClientBuilder;
+import org.apache.hc.core5.http.ClassicHttpResponse;
+import org.apache.hc.core5.http.io.entity.StringEntity;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import javax.annotation.Nullable;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+import static com.github.restdriver.clientdriver.RestClientDriver.giveResponse;
+import static com.github.restdriver.clientdriver.RestClientDriver.onRequestTo;
+import static org.apache.hc.core5.http.ContentType.TEXT_PLAIN;
+import static org.apache.hc.core5.http.HttpHeaders.CONTENT_TYPE;
+import static org.junit.jupiter.api.Assertions.*;
+
+class LogbookHttpExecHandlerTest extends AbstractHttpTest {
```

Comment:
The description of the PR is `If the response is compressed the Interceptor will log at the wrong time, thus use an ExecHandler`, but I don't see any test case that would cover this use-case. Can you maybe add a few test cases to demonstrate this behaviour?

Reply:
I added a testcase for to see the logging of an compressed response after decompression